### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by williamscch**
In this project, I implemented the CRUD (create, update, delete) methods to make all the elements fully functional in the UI.

Using modular JavaScript ES6 I build the following features for the web app:

- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
 - Implement a function for editing task descriptions.
- By default new tasks have the property completed set to false and the property index set to the value of the new array length.
- Deleting a task updates all remaining items' indexes, so they represent the current list order and are unique.
- All changes to the To-Do List are saved in local storage.

### CODE REVIEW

File: src/modules/module2.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        removeButton.classList.remove('active');
+      }
+    });
+
+    removeButton.addEventListener('click', () => {
+      removeButton.parentElement.remove();
+      tasksArray.splice(task.index, 1);
+      tasksArray.forEach((task, index) => {
+        task.index = index;
+      });
```
Comment (DeliceLydia) [inline src/modules/module2.js]: - [x] the property index should be set to the value of the new array length(i.e. if a 5th task is added to the list, the index of that task should equal to 5).

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @wdavidcch, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/modules/module2.js (hunk 1 above)
Comment (williamscch) [reply src/modules/module2.js]: Thank you so much, I will fix it.

**Review by Mre55 — APPROVED** [review]
**Hi @wdavidcch,**

# Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

Alt Text


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**